pkg/synopsysctl/cmd_update.go

```diff
@@ -624,11 +624,29 @@ func setBlackDuckFileOwnershipJob(namespace string, name string, pvcName string,
 			}
 			if job.Status.Succeeded > 0 {
 				log.Infof("successfully set the group ownership of files for PV '%s' in namespace '%s'", pvcName, namespace)
-				kubeClient.BatchV1().Jobs(job.Namespace).Delete(job.Name, &metav1.DeleteOptions{})
+
+				// Delete the Job
+				err = kubeClient.BatchV1().Jobs(job.Namespace).Delete(job.Name, &metav1.DeleteOptions{})
```

Comment:
Can you please set the delete options to have "propagationPolicy": "Background" and see the dependent pods are deleted?

Reply:
so that you don't need the following codes.